lib/server.rb

```diff
@@ -0,0 +1,34 @@
+require 'sinatra/base'
+require 'data_mapper'
+require './lib/peep.rb'
+require './lib/user.rb'
+require 'rack-flash'
+require 'sinatra/partial'
+
+require_relative './controllers/application'
+require_relative './controllers/peeps'
+require_relative './controllers/users'
+require_relative './controllers/sessions'
+require './lib/helpers/application'
+
+env = 'development'
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_challenge_#{env}")
+
```

Comment:
With this setup, your tests are all running against the development database, which means that your tests could potentially be polluted by development dummy data.